src/internal/components/pluggableVisualizations/bulletChart/bucketHelper.ts

```diff
@@ -0,0 +1,60 @@
+// (C) 2020 GoodData Corporation
+import * as BucketNames from "../../../../constants/bucketNames";
+import { IBucket, IBucketItem } from "../../../interfaces/Visualization";
+import { METRIC } from "../../../constants/bucket";
+import {
+    getPreferredBucketItems,
+    getAllAttributeItems,
+    limitNumberOfMeasuresInBuckets,
+} from "../../../utils/bucketHelper";
+
+const orderedMeasureBucketsLocalIdentifiers = [
+    BucketNames.MEASURES,
+    BucketNames.SECONDARY_MEASURES,
+    BucketNames.TERTIARY_MEASURES,
+];
+
+const placeMeasures = (buckets: IBucket[]) => {
```

Comment:
if we follow the naming of the main function, this could be `transformMeasureBuckets`